installation/managed-identity-authentication.md

```diff
@@ -0,0 +1,59 @@
+---
+description: This article explains how to upgrade your authentication to Managed Identity. 
+---
+
+# Managed Identity Authentication
+
+Instead of using SQL authentication, you are now able to **use Managed Identity authentication with SysKit Point**.
+
+In order to enable Managed Identity authentication to access data in the SysKit Point, complete the following:
+
+* **Navigate to [Azure Portal](https://portal.azure.com/) and open the SysKit Point resource group**
+* **Click the Managed Identity resource**; located under Resources and marked as Managed Identity in the Type column
+* **Copy the Client ID**; located on the right side of the Managed Identity resource screen 
+   * This will be needed later, so store the copied text somewhere it will be easy to get it later
+* Return back to the **SysKit Point resource group** page
+* **Click the Key Vault resource**; located under Resources and marked as Key Vault in the Type column
+* **Select the Access Policies option**; located on the left side of the screen
+* Click the **Create** button at the top of the screen
+  * This will provide the option to **Create an access policy**
+* Under **Configure from a template** select the **Secret Management** option and click on **Next**
+* For **Find Principal select the Admin** that is completing the Managed Identity authentication process and click **Next**
```

Comment:
ovdje bih stavio 'Find and select your account and click Next; with this you will gain permissions to perform changes in the key vault'